.env.example

```
@@ -80,3 +80,14 @@ TLSNOTARY_MAX_RECV_DATA=65536
 # ZK Identity System Configuration
 # Points awarded for each successful ZK attestation (default: 10)
 ZK_ATTESTATION_POINTS=10
+
+# ===========================================
+# Petri Consensus Configuration
+# ===========================================
+# Petri is enabled by default. Set to false to fall back to PoRBFT v2.
+PETRI_CONSENSUS=true
+PETRI_FORGE_INTERVAL_MS=2000
+PETRI_BLOCK_INTERVAL_MS=10000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PETRI_BLOCK_INTERVAL_MS key should go before the PETRI_CONSENSUS key

(UnorderedKey)